src/worker.js

```diff
@@ -217,6 +217,8 @@ function handleMessage(e) {
 #endif
 #endif // MODULARIZE && EXPORT_ES6
     } else if (e.data.cmd === 'run') {
+      Module['PThread'].receiveMessageRelayPort(e.data.port);
+      e.data.port.onmessage = handleMessage;
```

Comment:
So we use the same handler for messages regardless of where they come from?  interesting...